requirements.txt

```diff
@@ -4,10 +4,11 @@ cython==0.29.28
 dacite==1.6.0
 graphviz==0.20
 matplotlib==3.4.3
+networkx==2.6.3
 numpy==1.19.5
 pandas==1.3.5
 pydot==1.4.2
 robotoff==1.5.0
 scikit-learn==1.0.2
-tensorflow==2.6.3
```

Comment:
is there a specific reason to change the tensorflow version ?

Reply:
As it is a minor version, I think it is ok.